helper/functions.js

```diff
@@ -1,22 +1,27 @@
 const { parse, simplify } = require('prismarine-nbt');
 
 function titleCase(str) {
-  let splitStr = str.toLowerCase().replace(/_/g, ' ').split(' ');
-  for (let i = 0; i < splitStr.length; i++) {
-    if (!splitStr[i][0]) continue;
-    splitStr[i] = splitStr[i][0].toUpperCase() + splitStr[i].substr(1);
-  }
-  str = splitStr.join(' ');
-  return str;
+    let splitStr = str.toLowerCase().replace(/_/g, ' ').split(' ');
+    for (let i = 0; i < splitStr.length; i++) {
+        if (!splitStr[i][0]) continue;
+        splitStr[i] = splitStr[i][0].toUpperCase() + splitStr[i].substr(1);
+    }
+    str = splitStr.join(' ');
+    return str;
 }
 
-const decodeData = async (data) => {
-  const parsedNbt = await parse(Buffer.from(data, 'base64'));
-  const simplifiedNbt = simplify(parsedNbt.parsed);
-  return simplifiedNbt.i;
-};
+async function decodeData(data) {
+    const parsedNbt = await parse(Buffer.from(data, 'base64'));
+    const simplifiedNbt = simplify(parsedNbt.parsed);
+    return simplifiedNbt.i;
+}
```

Comment:
This could live in `ProfileNetworthCalculator`?

Reply:
we currently have ./helper/parseItems.js that is parsing the inventories and using the decodeData and as that one is rather large I would probably not put that in ProfileNetworthCalculator or what do you mean?